src/pages/login.jsx

```diff
@@ -0,0 +1,68 @@
+import "../styles/auth.css";
+import { Link } from "react-router-dom";
+import { useAuth } from "../context/authContext";
+import { useState } from "react";
+
+const Login = () =>{
+const { loginHandler } = useAuth();
+
+const [loginDetails, setLoginDetails] = useState({email : "", password : ""});
+
+
+const guestUserHandler = (event) => {
+event.preventDefault();
+setLoginDetails({
+email: "adarshbalika@gmail.com",
+password: "adarshBalika123",
```

Comment:
Previously, it was mentioned not to use adarshbalika login details as everyone was using it. You can change this in the db/users